packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -1028,7 +1028,7 @@ if (outcomeDetails.disputeRound < market.disputeRound) {
         outcome: outcomeValue.outcome,
         isInvalidOutcome: outcomeValue.invalid || false,
         isMalformedOutcome: outcomeValue.malformed || false,
-        bondSizeCurrent: new BigNumber(bondSizeCurrent).toFixed(),
+        bondSizeCurrent: new BigNumber(outcomeDetails.bondSizeCurrent || '0x0', 16).toFixed(),
```

Comment:
Did you figure out something upstream that caused (outcomeDetails.bondSizeCurrent to be undefined on newly disputed outcome in dispute round?

Reply:
Nope, just a bad merge!